src/modules/Builder/features/BuilderAppletSettings/IntegrationsListSetting/Integrations/LorisIntegration/DisconnectionPopup/DisconnectionPopup.tsx

```diff
@@ -19,7 +30,46 @@ export const DisconnectionPopup = ({ open, onClose }: DisconnectionPopupProps) =
       getScreens({
         onClose,
         setStep,
+        handleDisconnect: async () => {
+          try {
+            if (!appletId) {
+              return;
+            }
+
+            const res = await deleteIntegrationFromApi({
+              appletId,
+              integration_type: IntegrationTypes.Loris,
+            });
+
+            if (res.status === 204) {
+              const integrationsWithoutLoris = appletData?.integrations?.filter(
+                (i) => i.integrationType !== IntegrationTypes.Loris,
+              );
+
+              const appletUpdatedData = {
+                ...appletData,
+                integrations: integrationsWithoutLoris,
+              };
+
+              dispatch(updateAppletData(appletUpdatedData));
+              dispatch(
+                banners.actions.addBanner({
+                  key: 'SaveSuccessBanner',
+                  bannerProps: {
+                    children: t('loris.disconnectionSuccessful'),
+                    'data-testid': 'loris-disconnection-success-banner',
+                  },
+                }),
+              );
+            }
```

Comment:
I feel this entire section could be implemented at the Integrations level to allow storing and removing any integration and not just the specific type, but I'll leave that to your consideration